frontend/packages/ceph-storage-plugin/src/selectors/index.ts

```diff
@@ -1,5 +1,28 @@
 import * as _ from 'lodash';
 import { Alert } from '@console/internal/components/monitoring';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+
+const cephStorageProvisioners = ['ceph.rook.io/block', 'cephfs.csi.ceph.com', 'rbd.csi.ceph.com'];
+const cephStorageLabel = 'cluster.ocs.openshift.io/openshift-storage';
 
 export const filterCephAlerts = (alerts: Alert[]): Alert[] =>
   alerts.filter((alert) => _.get(alert, 'annotations.storage_type') === 'ceph');
+
+export const getFilteredPVCsData = (pvcsData: K8sResourceKind[]): K8sResourceKind[] =>
```

Comment:
i'd add default value
```suggestion
export const getFilteredPVCsData = (pvcsData: K8sResourceKind[] = []): K8sResourceKind[] =>
```